docs/tutorial/initial_setup.rst

```diff
@@ -0,0 +1,45 @@
+Initial Setup
+==================================================================
+
+.. code-block:: shell
+
+    lxc exec microceph-1 microceph cluster bootstrap
+    lxc exec microceph-1 microceph cluster add microceph-2
+    lxc exec microceph-1 microceph cluster add microceph-3
+
+The output of the above will look like:
+
+.. code-block:: shell
+
+    eyJuYW1lIjoibWljcm9jZXBoLTIiLCJzZWNyZXQiOiIwNWJkYzA3OTJkZmI4MDhjM2ZkMWJjNmIyMDFiMTUxMmI5NmY2N2QyNGEwMTdkZTFjMDNkOWIxZTBhZWFmZDI3IiwiZmluZ2VycHJpbnQiOiJkMjRkOGJiYjY0MTgyMWFlMjhkY2VlYWM2YmNkMGU4MmY1M2U2OTdmNDJjM2EyZTc0ZjhkMTk4MDhmNzZiNjgyIiwiam9pbl9hZGRyZXNzZXMiOlsiMTAuNjIuNzMuMTE0Ojc0NDMiXX0=
+    eyJuYW1lIjoibWljcm9jZXBoLTMiLCJzZWNyZXQiOiIyOWJhZGExZGYzYTMyYjJiNDRiZDIwYTliM2QwNGRiNzBjMjYzMTZlZjZmYjkzYTJhOTVkYjgzMWEwMmFjNGYwIiwiZmluZ2VycHJpbnQiOiJkMjRkOGJiYjY0MTgyMWFlMjhkY2VlYWM2YmNkMGU4MmY1M2U2OTdmNDJjM2EyZTc0ZjhkMTk4MDhmNzZiNjgyIiwiam9pbl9hZGRyZXNzZXMiOlsiMTAuNjIuNzMuMTE0Ojc0NDMiXX0=
+    
+Each line above is a token to be used, once, to join another node to the
+cluster, specifically microceph-2 and microceph-3, in order.
+
+.. code-block:: shell
+
+    lxc exec microceph-2 microceph cluster join eyJuYW1lIjoibWljcm9jZXBoLTIiLCJzZWNyZXQiOiIwNWJkYzA3OTJkZmI4MDhjM2ZkMWJjNmIyMDFiMTUxMmI5NmY2N2QyNGEwMTdkZTFjMDNkOWIxZTBhZWFmZDI3IiwiZmluZ2VycHJpbnQiOiJkMjRkOGJiYjY0MTgyMWFlMjhkY2VlYWM2YmNkMGU4MmY1M2U2OTdmNDJjM2EyZTc0ZjhkMTk4MDhmNzZiNjgyIiwiam9pbl9hZGRyZXNzZXMiOlsiMTAuNjIuNzMuMTE0Ojc0NDMiXX0=
+    lxc exec microceph-3 microceph cluster join eyJuYW1lIjoibWljcm9jZXBoLTMiLCJzZWNyZXQiOiIyOWJhZGExZGYzYTMyYjJiNDRiZDIwYTliM2QwNGRiNzBjMjYzMTZlZjZmYjkzYTJhOTVkYjgzMWEwMmFjNGYwIiwiZmluZ2VycHJpbnQiOiJkMjRkOGJiYjY0MTgyMWFlMjhkY2VlYWM2YmNkMGU4MmY1M2U2OTdmNDJjM2EyZTc0ZjhkMTk4MDhmNzZiNjgyIiwiam9pbl9hZGRyZXNzZXMiOlsiMTAuNjIuNzMuMTE0Ojc0NDMiXX0=
+
+You should now have a Ceph monitor cluster with quorum:
+
+.. code-block:: shell
+
+    $ lxc exec microceph-1 microceph.ceph status
+    cluster:
+        id:     a95eaf13-c3fe-466a-8278-d92186effbec
+        health: HEALTH_WARN
```

Comment:
I think reassuring readers that the `HEALTH_WARN` message is to be expected for now and that it should disappear once the OSD's are up.